guides/doc-Release_notes/topics/katello-4.4.0.adoc

```diff
@@ -0,0 +1,87 @@
+= Katello 4.4.0
+
+A full list of changes is available via https://projects.theforeman.org/projects/katello/issues?utf8=%E2%9C%93&set_filter=1&sort=id%3Adesc%2C%2C&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1517&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=[Redmine]
+
+= Katello 4.4.0 Habanero (2022-02-11)
```

Comment:
The release notes really should be something more curated rather than snippets from the CHANGELOG. The main reason being that the names of these things have no meaning to the user. And the average user isn't likely to be interested in the associated redmine and github information. IMO it's better to have a list of 5 meaningful things that a user can look to and say "yeah, that's a good reason to upgrade!" than 25 items that are virtually undecipherable. I know you were probably following whatever the last release engineer did, so I can't fault you there ;)


For my team, here are some notes (it's rough, feel free to adjust wording..)

- New interface available for syncing content between two Foreman servers. Available under the CDN Configuration tab in Content -> Subscriptions -> Manage Manifest.
- Red Hat repositories page no longer contacts the CDN when in Airgapped mode
- Numerous additions to the new host details UI including Installable Errata, Package filtering
- Numerous improvements to Content View exporting and importing
- Fixed errata installation raising errors when using Remote Execution

Reply:
> The release notes really should be something more curated rather than snippets from the CHANGELOG. The main reason being that the names of these things have no meaning to the user. And the average user isn't likely to be interested in the associated redmine and github information. IMO it's better to have a list of 5 meaningful things that a user can look to and say "yeah, that's a good reason to upgrade!" than 25 items that are virtually undecipherable. I know you were probably following whatever the last release engineer did, so I can't fault you there ;)

We have headline features for that. With Foreman we've always published headline features (which are for an entire x.y series) but also a detailed list of issues for every .z within that series. That's why I structured the documents as a `katello.adoc` and a `katello-x.y.z.adoc`.

Reply:
Oh, I should also mention that https://docs.theforeman.org/3.1/Release_notes/index-katello.html is what I'm talking about. You can see that first it also includes all of Foreman so users no longer need to read both guides. Also note that the releases are in chronological order.

It's not really complete, as you can see that [this part](https://docs.theforeman.org/3.1/Release_notes/index-katello.html#katello-release-notes) is empty.